examples/zephyr/fw_update/pytest/test_sample.py

```diff
@@ -0,0 +1,107 @@
+from golioth import Client
+import logging
+import pytest
+import time
+import yaml
+import datetime
+import os
+
+LOGGER = logging.getLogger(__name__)
+
+UPDATE_VERSION = '255.8.9'
+UPDATE_PACKAGE = 'main'
+API_URL = "https://api.golioth.io"
+CI_PROJ_NAME = "firmware_ci"
+
+pytestmark = pytest.mark.anyio
+
+async def test_fw_update(shell, device, credentials_file):
+    # Read credentials
+
+    with open(credentials_file, 'r') as f:
+        credentials = yaml.safe_load(f)
+
+    time.sleep(2)
+
+    # Add blueprint to device
+
+    board_name = os.environ['west_board']
+    LOGGER.info(f"Board name: {board_name}")
+
+    client = Client(api_url = API_URL, api_key = os.environ['GOLIOTH_API_KEY'])
+    project = await client.project_by_name(CI_PROJ_NAME)
```

Comment:
I just added a commit to test the alternative approach to requesting `project` from the fixture. This can be squashed once the CI is done running.